src/meta/Base.ts

```diff
@@ -15,15 +15,15 @@ export class Base extends Destroyable implements WidgetMetaBase {
 		this.nodeHandler = properties.nodeHandler;
 	}
 
-	public has(key: string): boolean {
+	public has(key: string | number): boolean {
 		return this.nodeHandler.has(key);
 	}
 
-	protected getNode(key: string): HTMLElement | undefined {
+	protected getNode(key: string | number): HTMLElement | undefined {
 		const node = this.nodeHandler.get(key);
 
 		if (!node && !this._requestedNodeKeys.has(key)) {
-			const handle = this.nodeHandler.on(key, () => {
+			const handle = this.nodeHandler.on(key as string, () => {
```

Comment:
Do you think that we should actually convert this to a string, instead of casting the TS type?
 ```
`${key}`
```

Reply:
Yeah, better than assuming there is some sort of implicit coercion of numbers downstream.